packages/ember-data/lib/system/relationship-meta.js

```diff
@@ -16,13 +16,24 @@ export function typeForRelationshipMeta(store, meta) {
   return type;
 }
 
+export function generateLink(store, kind, type) {
```

Comment:
yes if the `generateLink` function is found it will call the returned function every time but if the function is not found it is only called once.